jobsAdmin/JobsAdmin.go

```diff
@@ -335,7 +335,7 @@ func (ja *jobsAdmin) ResurrectJob(jobId common.JobID, sourceSAS string, destinat
 	files := func(prefix, ext string) []os.FileInfo {
 		var files []os.FileInfo
 		filepath.Walk(ja.planDir, func(path string, fileInfo os.FileInfo, _ error) error {
-			if !fileInfo.IsDir() && strings.HasPrefix(fileInfo.Name(), prefix) && strings.HasSuffix(fileInfo.Name(), ext) {
+			if !fileInfo.IsDir() && fileInfo.Size() != 0 && strings.HasPrefix(fileInfo.Name(), prefix) && strings.HasSuffix(fileInfo.Name(), ext) {
```

Comment:
Doing this check here will silently hide the issue and cause unexpected behavior.
Resurrect will succeed whereas we should fail.

Reply:
it will not succeed the resume job as this error will be caught [https://github.com/shubham808/azure-storage-azcopy/blob/d17f1f2d403d8b18d374795d03b37c3767d04c4f/jobsAdmin/init.go#L312](here). With this change we are treating empty files as missing, that should be sufficient or ?